frontend/apps/marketing/src/metadata/seo.ts

```diff
@@ -36,7 +36,9 @@ function getOpenGraph(
   locale: string,
 ) {
   const openGraphImage = seoMetadata.openGraphImage;
-  const openGraphImageUrl = getAbsoluteImageUrl(openGraphImage);
+  // As of July 2025, all open graph providers support JPEG & PNG but there is only partial support for AVIF.
+  // Use webp for compatibility.
+  const openGraphImageUrl = getAbsoluteImageUrl(openGraphImage, {fm: 'webp'});
```

Comment:
Do the params here need to be updated to match this pattern too? Want to make sure it won't break. https://github.com/code-dot-org/code-dot-org/blob/880f884acc51c5461ce670c16cd9a5e40298e712/frontend/apps/marketing/src/components/contentful/collections/peopleCollection/PeopleCollection.tsx#L97

Reply:
I updated the API to both formats